src/omnicore/rules.cpp

```diff
@@ -72,6 +72,7 @@ std::vector<TransactionRestriction> CConsensusParams::GetRestrictions() const
         { MSC_TYPE_SEND_ALL,                  MP_TX_PKT_V0,  false,   MSC_SEND_ALL_BLOCK },
 
         { MSC_TYPE_OFFER_ACCEPT_A_BET,        MP_TX_PKT_V0,  false,   MSC_BET_BLOCK      },
+        { MSC_TYPE_DEX_PAYMENT,               MP_TX_PKT_V0,  true,    MSC_METADEX_BLOCK }
```

Comment:
Is DEX payment not supported in the original OmniLyer?

Reply:
It's supported but in the old fashion way (using Class B transactions ) spending dust in Exodus address.